_includes/devdoc/bitcoin-core/rpcs/rpcs/listtransactions.md

```diff
@@ -188,7 +203,9 @@ Result:
         "walletconflicts" : [
         ],
         "time" : 1418695703,
-        "timereceived" : 1418925580
+        "timereceived" : 1418925580,
```

Comment:
The trusted field is only shown when the transaction is still unconfirmed. I didn't have an unconfirmed tx in my wallet.